### ISSUE TITLE
Calling/creating an actor from DaprClient

### ISSUE THREAD

**Issue by florimm** [dapr/js-sdk#161]
## Expected Behavior

I'm using the latest's framework 2.0.0 and was trying to call an actors through daprClient as in examples that are in the repo but it seems like Its not accessible.

## Actual Behavior
As you see in print-scr actor does not exist and I get undefined if I try to call it
![image](https://user-images.githubusercontent.com/1766825/151325133-b82f486a-2f3d-4b89-988d-21f88957c003.png)

## Steps to Reproduce the Problem
Try to create a new DaprClient and call actors 
```
const client = new DaprClient(daprConfig.daprHost, daprConfig.daprPort);
client.actor.invoke...
```
actor is undefined

**Comment by XavierGeerinck** [comment]
Hi @florimm ,

Thanks a lot for reporting this. Actors were introduced and pulled out of the main SDK. It would however for developer reasons be more friendly to have them accessible in this way as well.

I added it to the list and am working on it :)

**Comment by XavierGeerinck** [comment]
Hi @florimm ! 

Finished the PR for this and you should be able to call it now through the Client interaction.

Note that his will be available starting from `2.1.0` which I hope to release later today. 

https://github.com/dapr/js-sdk/blob/master/CHANGELOG.md#v210 explains in more detail how you can use it now. Let me know if this helped you!
